tests/integration_tests/modules/test_ubuntu_advantage.py

```diff
@@ -147,7 +147,6 @@ def install_ua_daily(session_cloud: IntegrationCloud):
         client.destroy()
 
 
-@pytest.mark.adhoc
```

Comment:
Seeing failures on azure with this PR currently. We are hitting autoattach errors from that full_auto_attach_api call. We may need to make sure we update and install latest released ubuntu-advantage-tools as maybe I'm dealing with stale jammy PRO images.
```
CLOUD_INIT_PLATFORM=azure CLOUD_INIT_OS_IMAGE=jammy  CLOUD_INIT_CLOUD_INIT_SOURCE=./cloud-init_22.3-55-g3e63d9c2-1~bddeb_all.deb .tox/integration-tests/bin/pytest tests/integration_tests/modules/test_ubuntu_advantage.py::TestUbuntuAdvantagePro::test_custom_services
...

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/cloudinit/config/cc_ubuntu_advantage.py", line 397, in _auto_attach
    full_auto_attach(options=options)
  File "/usr/lib/python3/dist-packages/uaclient/api/u/pro/attach/auto/full_auto_attach/v1.py", line 64, in full_auto_attach
    return _full_auto_attach(options, UAConfig(root_mode=True))
  File "/usr/lib/python3/dist-packages/uaclient/api/u/pro/attach/auto/full_auto_attach/v1.py", line 78, in _full_auto_attach
    raise e
  File "/usr/lib/python3/dist-packages/uaclient/api/u/pro/attach/auto/full_auto_attach/v1.py", line 75, in _full_auto_attach
    ret = _full_auto_attach_in_lock(options, cfg)
  File "/usr/lib/python3/dist-packages/uaclient/api/u/pro/attach/auto/full_auto_attach/v1.py", line 88, in _full_auto_attach_in_lock
    raise exceptions.AlreadyAttachedError(
uaclient.exceptions.AlreadyAttachedError: 12aad61c-6de4-4e53-a6c6-5aff52a83777

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/cloudinit/config/modules.py", line 246, in _run_modules
    ran, _r = cc.run(
  File "/usr/lib/python3/dist-packages/cloudinit/cloud.py", line 67, in run
    return self._runners.run(name, functor, args, freq, clear_on_fail)
  File "/usr/lib/python3/dist-packages/cloudinit/helpers.py", line 185, in run
    results = functor(*args)
  File "/usr/lib/python3/dist-packages/cloudinit/config/cc_ubuntu_advantage.py", line 445, in handle
    _auto_attach(ua_section)
  File "/usr/lib/python3/dist-packages/cloudinit/config/cc_ubuntu_advantage.py", line 401, in _auto_attach
    raise RuntimeError(msg) from ex
RuntimeError: Error during `full_auto_attach`: 12aad61c-6de4-4e53-a6c6-5aff52a83777
```